python/ray/data/tests/test_dynamic_block_split.py

```diff
@@ -82,81 +84,87 @@ def test_enable_in_ray_client(ray_start_cluster_enabled):
     ],
 )
 def test_dataset(
-    ray_start_regular_shared,
+    shutdown_only,
     enable_dynamic_block_splitting,
     target_max_block_size,
     compute,
 ):
-    # Test 10 blocks from 10 tasks, each block is 1024 bytes.
-    num_blocks = 10
+    ray.shutdown()
+    # We need at least 2 CPUs to run a actorpool streaming
+    ray.init(num_cpus=2)
```

Comment:
Just to double check - is this the real code change to fix test failure, besides the renaming from `num_blocks_per_task` and `num_blocks`?

Reply:
Yes. Renaming is no-op other than readability.